src/prefix.js

```diff
@@ -202,13 +203,11 @@ class WebAdapter {
             }
         }, options));
     }
-    config(name, value) {
-        if (typeof name === 'string') {
-            this._config[name] = value;
-        } else {
-            const { completion, ...rest } = name;
-            this._term.settings().completion = completion;
-            $.extend(rest, name);
+    config(name) {
+        if (typeof name === 'object') {
+            for (const [key, value] of Object.entries(name)) {
+                this._config[key] = value
+            }
```

Comment:
You remove old behavior and you deleted `completion` option. Now all options will stop working since you only set this._config and ignore terminal settings.